mapiproxy/servers/default/emsmdb/emsmdbp_object.c

```diff
@@ -3129,3 +3132,218 @@ _PUBLIC_ struct emsmdbp_object *emsmdbp_object_ftcontext_init(TALLOC_CTX *mem_ct
 
 	return object;
 }
+
+static enum MAPISTATUS emsmdbp_object_sharing_metadata_property(struct emsmdbp_context *emsmdbp_ctx, struct emsmdbp_object *sharing_object, struct Binary_r **attach_bin, TALLOC_CTX *mem_ctx)
```

Comment:
Can you add a doxygen comment explicitly mentioning that the caller function is responsible for calling talloc_free on mem_ctx passed to the function in order to avoid mem leaks.